### PR TITLE
Kaushan

### PR DESCRIPTION
**Pull request by HarishDass**

### CODE REVIEW

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -1,2 +1,21 @@
-<button (click)="navigateTO('EmployeeDetails')">EmployeeDetails</button>
-<button  (click)="navigateTO('Attendance')">Attendance</button>
+<div id="navBar">
+    <img src="../../assets/Images/1_11zon-fotor-2023090111414-removebg-preview (1).png" alt="Employee Tracker">
+    <p-button label="Log out" styleClass="p-button-rounded"></p-button>
+</div>
+<p-divider></p-divider>
+
+<div id="mainView">
+    <p-card header="" class="Card" [style]="{ width: '360px',height:'250px'}" (click)="navigateTO('EmployeeDetails')">
```
Comment (HarishDass) [inline src/app/home/home.component.html]: Do not use in-line stykes

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+        <div class="cardinDiv">Employee Details</div>
+        <ng-template pTemplate="header">
+            <img alt="Card" src="../../assets/Images/AdobeStock_427200891 .png" />
+        </ng-template>
+    </p-card>
+    <p-card header="" class="Card" [style]="{ width: '360px',height:'250px',}" (click)="navigateTO('Attendance')">
```
Comment (HarishDass) [inline src/app/home/home.component.html]: Do not use in-line stykes

File: src/app/home/home.component.html [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+        <div class="cardinDiv">Attendance</div>
```
Comment (HarishDass) [inline src/app/home/home.component.html]: Please use the naming conventions in class

File: src/app/home/home.component.html [hunk 4]
```diff
@@ -1,2 +1,21 @@
...
+        <ng-template pTemplate="header">
+            <img alt="Card" src="../../assets/Images/Candidate-1024x585.jpg" style="height: 174px;" />
```
Comment (HarishDass) [inline src/app/home/home.component.html]: Do not use inline styles

File: src/app/home/home.component.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+#mainView {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: last baseline;
+    margin: 90px auto;
+}
+
+.Card {
```
Comment (HarishDass) [inline src/app/home/home.component.css]: Use the proper naming convention

File: src/app/home/home.component.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    cursor: pointer;
+    margin-top: 20px;
+    text-align: center;
+    transition: 1s;
+}
+
+.Card:hover {
```
Comment (HarishDass) [inline src/app/home/home.component.css]: Use the proper naming convention

File: src/app/home/home.component.css [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    transform: scale(1.1);
+}
+
+.cardinDiv {
```
Comment (HarishDass) [inline src/app/home/home.component.css]: Use the proper naming convention

File: src/app/home/home.component.css [hunk 8]
```diff
@@ -0,0 +1,33 @@
+#navBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: rgba(255, 255, 255, 0.884);
+}
+
+#mainView {
```
Comment (HarishDass) [inline src/app/home/home.component.css]: Use the proper naming convention

File: src/app/home/home.component.html [hunk 9]
```diff
@@ -1,2 +1,35 @@
...
+    header=""
+    class="employee_details_card"
+    (click)="navigateTO('Attendance')"
+  >
+    <div class="employe_card_text">Attendance</div>
+    <ng-template pTemplate="header">
+      <img
+        alt="Card"
+        src="../../assets/Images/Candidate-1024x585.jpg"
+        style="height: 174px"
```
Comment (HarishDass) [inline src/app/home/home.component.html]: don't use inline style

File: src/app/home/home.component.html [hunk 10]
```diff
@@ -1,2 +1,35 @@
...
+    src="../../assets/Images/1_11zon-fotor-2023090111414-removebg-preview (1).png"
+    alt="Employee Tracker"
+  />
+  <p-button label="Log out" styleClass="p-button-rounded"></p-button>
+</div>
+<p-divider></p-divider>
+
+<div id="mainView">
+  <p-card
+    header=""
```
Comment (HarishDass) [inline src/app/home/home.component.html]: remove the header

File: src/app/home/home.component.html [hunk 11]
```diff
@@ -1,2 +1,35 @@
...
+    class="employee_details_card"
+    (click)="navigateTO('EmployeeDetails')"
+  >
+    <div class="employe_card_text">Employee Details</div>
+    <ng-template pTemplate="header">
+      <img alt="Card" src="../../assets/Images/AdobeStock_427200891 .png" />
+    </ng-template>
+  </p-card>
+  <p-card
+    header=""
```
Comment (HarishDass) [inline src/app/home/home.component.html]: remove the header

File: src/app/home/home.component.css [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.employee_details_card:hover {
+    transform: scale(1.1);
+}
+
+.employe_card_text{
+    font-weight: bold;
+    position: relative;
+    /* bottom: 20px; */
```
Comment (HarishDass) [inline src/app/home/home.component.css]: remove this line

File: src/app/home/home.component.css [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.employee_details_card {
+    cursor: pointer;
+    margin-top: 20px;
+    text-align: center;
+    transition: 1s;
+    width: 360px;
+    height:250px;
+    /* [style]="{ width: '360px',height:'250px */
```
Comment (HarishDass) [inline src/app/home/home.component.css]: remove this line

**Review by HarishDass — COMMENTED** [review]
finish them

File: src/app/employee-details/inactive-employee/inactive-employee.component.ts [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+  styleUrls: ['./inactive-employee.component.css']
+})
+export class InactiveEmployeeComponent {
+  leftoverdatas:EmployeData[]=[]
+
+  constructor(private LinkData:LeftoverTableService){}
+  ngOnInit()
+  {this.LinkData.getDataFn().subscribe((res:EmployeData[])=>{
+      this.leftoverdatas=res
+      // console.log(this.getData)
```
Comment (HarishDass) [inline src/app/employee-details/inactive-employee/inactive-employee.component.ts]: remove this

**Comment by Kaushan05** [comment]
home page done every naming conventions are changed correctly

**Pull request merged**